src/main/java/com/epam/ta/reportportal/ws/model/analyzer/IndexLaunch.java

```diff
@@ -20,6 +20,8 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.time.LocalDate;
```

Comment:
Please remove unused imports.
Also please(for future) name branch with postfix with some small description.